README.md

```diff
@@ -108,6 +108,20 @@ To integrate SwiftLint with [Atom](https://atom.io/), install the
 [`linter-swiftlint`](https://atom.io/packages/linter-swiftlint) package from
 APM.
 
+### fastlane
+
+You can use the [official swiftlint fastlane action](https://docs.fastlane.tools/actions/#swiftlint) to run SwiftLint as part of your fastlane process.
+
+```ruby
+swiftlint(
+  mode: :lint,                          # SwiftLint mode: :lint (default) or :autocorrect
+  output_file: "swiftlint.result.json", # The path of the output file (optional)
```

Comment:
what do you think about adding `reporter` too so people know that it's not just a matter of using `.json`?

I also think adding `executable` is important since integration via CocoaPods is now available.